packages/build-ui/src/pages/agits/_id/management/artists/_id/page.rs

```diff
@@ -346,7 +346,22 @@ pub fn NewArtistPage(lang: Language, agit_id: ReadOnlySignal<i64>) -> Element {
 
                     button {
                         class: "border border-white text-white px-6 py-2 flex items-center justify-center",
-                        // This would navigate to add artwork page
+                        onclick: move |_| {
+                            tracing::debug!(
+                                "Display name: {:?}, social media: {:?}, introduction: {:?}, biography: {:?}",
+                                display_name.read().clone(),
+                                social_media.read().clone(),
+                                introduction.read().clone(),
+                                biography.read().clone(),
+                            );
+                                ctrl.create_artist(
+                                 display_name.read().clone(),
+                                 display_name.read().clone(),
+                                 social_media.read().clone(),
+                                 introduction.read().clone(),
+                                 biography.read().clone(),
+                                );
+                        },
```

Comment:
```suggestion
                        onclick: move |_| {
                                ctrl.create_artist(
                                 display_name.read().clone(),
                                 display_name.read().clone(),
                                 social_media.read().clone(),
                                 introduction.read().clone(),
                                 biography.read().clone(),
                                );
                        },
```

move logs and all logics to handler of controller.